### PR TITLE
panel_19

### LINKED ISSUE #287: panel_19
Description: Create a 2 by 2 grid of axes, then show how to adjust the positioning of the axes

NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/panel_19.ncl
NCL Plots: https://www.ncl.ucar.edu/Applications/Images/panel_19_1_lg.png
https://www.ncl.ucar.edu/Applications/Images/panel_19_2_lg.png
https://www.ncl.ucar.edu/Applications/Images/panel_19_3_lg.png 

Category: Panel

### PR DESCRIPTION
**Pull request by hCraker**
Closes #287 
Current state of plot:
image

Do to:
- add titles and lables
- add colorbar
- create examples with different spacing between subplots

### CODE REVIEW

**Comment by hCraker** [comment]
Current state of the three figures

image
image
image

File: Plots/Panels/NCL_panel_19.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+"""
+NCL_panel_19.py
+===============
+This script illustrates the following concepts:
+   - Paneling four subplots in a two by two grid using ``gridspec``
+   - Adjusting the positioning of the subplots using ``hpace`` and ``wspace``
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_19.py:6]:
```suggestion
   - Adjusting the positioning of the subplots using ``hspace`` and ``wspace``
```
Spelling

**Review by michaelavs — COMMENTED** [review]
So far everything looks good and is running correctly on my side! I did notice that for each plot call, the code was virtually identical. So would it be possible to create a single function to call instead of repeating the three code blocks?

**Review by michaelavs — APPROVED** [review]
Looks good, thanks for making those changes!

**Review by anissa111 — APPROVED** [review]
This looks great, Rose!

**Pull request merged**